Makefile

```diff
@@ -0,0 +1,17 @@
+first_start: # Начальная инициализация
```

Comment:
А этот файл нужен для работы команд в консоли? 
Я использовал его когда-то только для сборки проекта на c++, но было давно и уже не помню
И он только для линукса? под винду нельзя тоже адаптировать? может какую-то проверку добавлять

Reply:
На винде можно, но через костыли. Погуглю, может что нового появилось.
Вроде ещё на маке можно, но мне не на чем проверить.
Юзается просто - в консоле 'make first_start' к примеру.
Я обычно использую как сокращатель для длинных или часто повторяющихся команд.

Reply:
Нашел относительно простой способ поставить на винду:
```
Alternatively, if you’d prefer to use the new native Windows package manager you can use this winget command:

>> winget install GnuWin32.Make

This will download the installer and run it for you automatically. Make will also be added to the list of installed programs so you can uninstall it from the Add or remove programs section. You will still get the following error since the installer does not add make to the system path:

make : The term 'make' is not recognized as the name of a cmdlet, function, script file, or operable program...

To make it executable you need to open the Start menu and search for Edit the system environment variables. Click Environment Variables, then under System variables choose Path and click Edit. Click New and insert C:\Program Files (x86)\GnuWin32\bin, then save the changes. Open a new terminal and test that the command works.
```
Могу расписать в ридми как сделать, и после добавим команд.

Reply:
Пока уберу make, очень странно на винде работает. Если получится подружить - предложу отдельно.